website/docs/syncing_metastore.md

```diff
@@ -22,10 +22,65 @@ cd hudi-hive
  [hudi-hive]$ ./run_sync_tool.sh --help
 ```
 
+### Hive Sync Configuration
 
-## Flink Setup
+Please take a look at the arguments that can be passed to `run_sync_tool` in [HiveSyncConfig](https://github.com/apache/hudi/blob/master/hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java).
+Among them, following are the required arguments:
+```java
+@Parameter(names = {"--database"}, description = "name of the target database in Hive", required = true);
+@Parameter(names = {"--table"}, description = "name of the target table in Hive", required = true);
+@Parameter(names = {"--base-path"}, description = "Basepath of hoodie table to sync", required = true);## Sync modes
+```
+Corresponding datasource options for the most commonly used hive sync configs are as follows:
+
+| HiveSyncConfig | DataSourceWriteOption | Description |
```

Comment:
Can you also add a link to DatasourceWriteOptions somewhere incase some is interested to look at datasource config files where hive sync configs are present.